flow/src/main/java/flow/Flow.java

```diff
@@ -65,6 +65,11 @@
     return wrapper.services.getService(serviceName);
   }
 
+  /** @return null if context does not contain the named service. */
+  @Nullable public static <T> T getService(@NonNull String serviceName, @NonNull View view) {
```

Comment:
I don't like  this. It's barely a convenience, and trivial to implement yourself if you really want it. Doesn't belong in the library.